astroquery/eso/core.py

```diff
@@ -207,6 +207,8 @@ def list_instruments(self, cache=True):
                     if instrument not in self._instrument_list:
                         self._instrument_list.append(instrument)
             self._instrument_list.append(u'harps')
+            self._instrument_list.append(u'ambient_paranal')
+            self._instrument_list.append(u'meteo_paranal')
```

Comment:
Weird, but it seems that page is very much VLT (or at least Paranal) biased. HARPS is then very much an outlier there.